.ci/Manifest.toml

```diff
@@ -36,11 +36,11 @@ version = "0.4.0"
 
 [[BinaryBuilderBase]]
 deps = ["CodecZlib", "Downloads", "InteractiveUtils", "JSON", "LibGit2", "Libdl", "Logging", "OutputCollectors", "Pkg", "Random", "SHA", "Scratch", "SimpleBufferStream", "TOML", "Tar", "UUIDs", "p7zip_jll", "pigz_jll"]
-git-tree-sha1 = "225b107285e25a745047582d138cd99986f52f1a"
```

Comment:
Next time you update this pull request you can revert this change, as it is now live here